README.md

```diff
@@ -1,4 +1,5 @@
-GIT Documentation Translations
+# GIT Documentation Translations
+
 =====================
```

Comment:
We should probably also drop these asciidoc headline markers if we change the readme to markdown.

Reply:
Good question. How does it render with asciidoc? if it's ok, let's just rename the file.

Reply:
Replying to myself, this document is clearly not asciidoc, but that would still make sense to move it to asciidoc instead of markdown, all the other documents already being in asciidoc.

Reply:
Does it make sense, though?

We don't build and host a rendered version of this file and probably never will do. It's really mainly a README for people new to this repo. And having people build their own README from said README seems a little overly convoluted to me.

Reply:
I was implying that people should build the README file! GitHub can render asciidoc files directly if their suffix is `.adoc`.

Reply:
Oh. That makes more sense.